.github/workflows/update-durations.yml

```diff
@@ -0,0 +1,84 @@
+name: Update Durations Files
+on:
+  # Scheduled trigger every Saturday at 2:47am UTC
+  schedule:
+  - cron: '47 2 * * 6'
+  workflow_dispatch:
```

Comment:
I added a `workflow_dispatch` trigger so that we can manually trigger the workflow if any PR massively imbalances the load.